components/brave_ads/core/internal/user_engagement/conversions/conversions_test_base.cc

```diff
@@ -33,12 +36,14 @@ void BraveAdsConversionsTestBase::TearDown() {
 
 void BraveAdsConversionsTestBase::VerifyOnDidConvertAdExpectation(
     const AdInfo& ad,
-    ConversionActionType action_type) {
+    ConversionActionType action_type,
+    base::OnceClosure did_convert_closure) {
```

Comment:
nit: `did_convert_ad_closure`, and the same for other `VerifyOn*Expectation`. Thanks

Reply:
Fixed